src/sentry/static/sentry/app/views/eventsV2/miniGraph.tsx

```diff
@@ -30,12 +25,33 @@ class MiniGraph extends React.Component<Props> {
     // than the cost for rendering the graph, which can take ~200ms to ~300ms to
     // render.
 
-    return !isEqual(omitProps(this.props), omitProps(nextProps));
+    return !isEqual(this.getRelevantProps(this.props), this.getRelevantProps(nextProps));
+  }
+
+  getRelevantProps(props: Props) {
```

Comment:
Could call this `getRefreshProps` to be clearer that the subset is used to refresh data. :bike:

Reply:
🚲